cmake/std/atdm/README.md

```diff
@@ -432,3 +457,5 @@ they support are:
   `shiller` and the mirror SON machine `hansen`.
   
 * `toss3/`: Supports SNL HPC machines `serrano/` and `chama/`.
+
+* `mutrino/`: Supports SNL HPC machine `mutrino/`.
```

Comment:
It would be nice if we could list these directories in alphabetical order.  Can we make that change?